xchange-core/src/main/java/org/knowm/xchange/dto/account/OpenPosition.java

```diff
@@ -78,13 +131,21 @@ public String toString() {
     private Type type;
     private BigDecimal size;
     private BigDecimal price;
+    private BigDecimal markPrice;
+    private BigDecimal leverage;
+    private BigDecimal marginRatio;
+    private Date timestamp;
 
     public static Builder from(OpenPosition openPosition) {
```

Comment:
could you please add @JsonPOJOBuilder(withPrefix = "") in order to support json deserialisation that is needed in nostro persistency module

also @JsonDeserialize(builder = OpenPosition.Builder.class) in the top of OpenPosition class

Reply:
Fixed, serialization test added